conjur/util/parser_utils.py

```diff
@@ -0,0 +1,77 @@
+"""
+This Module holds all the utils needed for the ArgParseBuilder
+"""
+
+import argparse
+
+
+def formatter(prog):
```

Comment:
Can the methods be declared as static?

Reply:
these are module functions (contextless free functions). no need to use the @staticMethod decorator as they are not part of no class.